Packs/CommonScripts/Scripts/GenerateAsBuilt/GenerateAsBuilt.py

```diff
@@ -884,5 +961,23 @@ def main():
     return_results(fr)
 
 
+def main():  # pragma: no cover
+    args = demisto.args()
+    author = args.get("author")
+    customer = args.get("customer")
+
+    # Given a playbook is passed, we generate a use case document, instead of the platform as build.
+    if playbook := args.get("playbook"):
+        playbook_use_case(playbook, author, customer)
+        return
```

Comment:
No need when you have no line to continue to.

Reply:
fixed